cmd/sovereignnode/sovereignNodeRunner.go

```diff
@@ -84,11 +88,11 @@ const (
 
 // sovereignNodeRunner holds the sovereign node runner configuration and controls running of a node
 type sovereignNodeRunner struct {
-	configs *config.Configs
+	configs *sovereignConfig.SovereignConfig
```

Comment:
sovereignConfig

Reply:
I would leave this as it is. This would cause ~80 changes. Since `configs` is of type ` *sovereignConfig.SovereignConfig`, I think it's not confusing

Reply:
ok